custom_typings/globals.d.ts

```diff
@@ -0,0 +1,12 @@
+interface Event {
+  composedPath(): EventTarget[];
```

Comment:
I think as of TypeScript 3.0 this type is in the built-in typings

Reply:
`EventInit.composed` was available, but not `composedPath()`:

```
src/router.ts:52:22 - error TS2339: Property 'composedPath' does not exist on type 'MouseEvent'.

52     const anchor = e.composedPath().filter(
                        ~~~~~~~~~~~~
```

Reply:
Yeah `composedPath()` is not in the current 3.0 release yet, but should be in the next one.
https://github.com/Microsoft/TSJS-lib-generator/pull/523